app/controllers/api/v1/application_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class ApplicationController < ActionController::API
```

Comment:
Top access API, the client first need to access `client_id`, `secret`.  How s/he will get it?

In our Go certification, we mostly [created](https://github.com/Lahphim/go-crawler-challenge/pull/69) a view, where logged in user can generate the `client_id` and `secret`.  For Doorkeeper, it does everything. You might need to change the route setting or so. What do you think?

Reply:
I think it depends on the specifications (or say, the client needs) :) 
Here, the IC scenario says that WE are going to build a mobile app out of that, so my assumption is that the Client Id does not need public endpoints. 

But let's be player, I will update the code as if we are in a situation where other people/organizations might want to consume ou API. ;-)

Reply:
All right, interesting adventure :) It's done & I have also research on how to manually create/edit applications using Doorkeeper library (might be useful for future projects). 

The description of this change has been added in the PR description ;-)

Thanks!